src/MICore/LaunchOptions.cs

```diff
@@ -1579,9 +1571,9 @@
             {
                 DebugChildProcesses = suppOptions.DebugChildProcesses;
             }
-            if (string.IsNullOrWhiteSpace(VisualizerFile))
+            if (VisualizerFiles == null || VisualizerFiles.Count == 0)
```

Comment:
>VisualizerFiles == null ||  [](http://example.com/codeflow?start=16&length=27)

nit: no need to check for null anymore

Reply:
This specific instance is for the XML merge case where its using the XML.VisualizerFile not the JSON one. 

I would remove the outer loop gregg suggested and just do the inner check with `file` as suppOptions.VisualizerFile